x/common/testutil/testapp/test_util.go

```diff
@@ -93,7 +94,9 @@ func genesisStateWithValSet(
 		delegations = append(delegations, stakingtypes.NewDelegation(genAccs[0].GetAddress(), val.Address.Bytes(), math.LegacyOneDec()))
 	}
 	// set validators and delegations
-	stakingGenesis := stakingtypes.NewGenesisState(stakingtypes.DefaultParams(), validators, delegations)
+	stakingParams := stakingtypes.DefaultParams()
+	stakingParams.BondDenom = appconst.BondDenom
+	stakingGenesis := stakingtypes.NewGenesisState(stakingParams, validators, delegations)
```

Comment:
I needed the stake denom to be `unibi` instead of `stake` for the wasm precompile test that delegates.